src/Workspaces/Core/Portable/CodeGeneration/AbstractCodeGenerationService.cs

```diff
@@ -262,7 +262,10 @@ protected static T Cast<T>(object value)
                 // Metadata as source generates complete declarations and doesn't modify
```

Comment:
We do use this code to generate enum members, but with auto insertion location = true.